README.md

```diff
@@ -35,4 +35,4 @@ Create a [new GitHub release](https://github.com/hypothesis/wp-hypothesis/releas
 
 ## License
 
-[BSD](http://opensource.org/licenses/BSD-2-Clause)
+[BSD-2-Clause](http://opensource.org/licenses/BSD-2-Clause)
```

Comment:
`BSD-2-Clause` is the proper [SPDX identifier](https://spdx.org/licenses/), so let's try this. I saw another plugin use `BSD-2` instead though (https://github.com/gumlet/wordpress-plugin/blob/f8e13074e3cfab72635d75b912e3d681a7f95d99/readme.txt#L12).